ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -58,6 +60,29 @@ StatusSectionLayout {
     property var assetsModel
     property var collectiblesModel
 
+    function filterPendingPermissions(model) {
+        return !!model && model.permissionState !== PermissionTypes.State.Approved
+    }
+
+    readonly property var pendingViewOnlyPermissionsModel: SortFilterProxyModel {
+            sourceModel: root.viewOnlyPermissionsModel
+        filters: [
+            ExpressionFilter {
+                expression: root.filterPendingPermissions(model)
```

Comment:
IIRC ui-team said that `ExpressionRole` is the slowest one.
The condition seems to be quite an easy one, could we improve this code? 🤔 

cc @micieslak

Reply:
`FastExpressionFilter` at least :) same below

Reply:
Or, you could do the model with `ValueFilter` + `inverted`, and then `.count` the items in the model

Reply:
And also, there's this code in chat_content module:
```nim
method getPermissionsCheckOngoing*(self: Module): bool =
  self.view.getPermissionsCheckOngoing()
```
just not exposed as property; should be easy to add that instead?

Reply:
Updated. I used the `ValueFilter` as recommended and it worked well.

As for using `getPermissionsCheckOngoing`, it wouldn't work in this case, because it's not about the permissions being checked if they are respected, it's about the permission being edited by the admin. Until the control node receives the edit and applies it, the channel will be blocked for the admin. 

In case anyone is wondering, other people can still use it, because the permission change didn't reach them yet so the old encryption key (or rather the lack of it if it was open) is still valid.
In theory, we could make it so that the admin can still post, but that requires some hard changes on the backend to check if the permission is updated and not applied yet and if so, to use the old encryption key instead. It's not worth the time and effort. I might open an issue to track it though.